src/payment/strategies/paypal-commerce/paypal-commerce-funding-key-resolver.ts

```diff
@@ -18,7 +18,7 @@ export default class PaypalCommerceFundingKeyResolver {
                     return 'BANCONTACT';
                 case 'giropay':
                     return 'GIROPAY';
-                case 'przelewy24':
+                case 'p24':
```

Comment:
This seems like a contract change in the API. Is this method live? if so, will this break current consumers?

Reply:
@icatalina backend guys have this fix too